File: Modul 4/Soal 4/PRAK304-2210817310006-Ahmad Muhsin Zein.py
i = 0
while True:
    i = i + 1
    if(i>5):
        break
    a=int(input())
    if(a >= 1 and a <= 9):
        print("Satuan\n")
    elif(a < 1 and a > -1):
        print("nol\n")
    elif(a > 99):
        print("Anda menginput Melebihi Limit Bilangan\n")
    elif(a >= 20 and a <= 99):
        print("Puluhan\n")
    elif(a >= 11 and a <= 19):
        print("Belasan\n")

File: Modul 4/Soal 4/PRAK304-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

int main()
{
    int a;
    for(int i = 0; i <= 4; i ++){
        scanf("%d", &a);
        if(a >= 1 && a <= 9){
            printf("Satuan\n\n");
        }
        else if(a < 1 && a > -1){
            printf("Nol\n\n");
        }
        else if(a > 99){
            printf("Anda Menginput Melebihi Limit Bilangan\n\n");
        }
        else if(a >= 20 && a <= 99){
            printf("Puluhan\n\n");
        }
        else if(a >= 11 && a <= 19){
            printf("Belasan\n\n");
        }
    }
}
